test/data/webhooks/webhook.tag_push.json

```diff
@@ -0,0 +1,10 @@
+{"headers": {"Host": "requestb.in",
+"Accept": "*/*",
+"User-Agent": "GitHub-Hookshot/6f33334",
+"X-GitHub-Event": "push",
+"X-GitHub-Delivery": "7639a9808fq-11w3-2359-6266dd99y98d",
+"content-type": "application/json",
+"X-Hub-Signature": "sha1=14b935ba703e45e7c6d64f3cc7e961d85e499015",
```

Comment:
I have to change `X-Hub-Signature` so that it passes [this](https://github.com/taskcluster/taskcluster-github/blob/master/src/api.js#L207) condition in src/api.js
```
    if (!webhookSecrets.some(webhookSecret => {
      let calculatedSignature = generateXHubSignature(webhookSecret,
        JSON.stringify(body));
      return compareSignatures(calculatedSignature, xHubSignature);
```